CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
```

Comment:
change the with-next example in the examples folder to use the new functions. Also make sure that the new functions work as expected in a nextjs app (with session refreshing and all working as expected)

Reply:
done